src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -36,7 +36,7 @@
     <!-- DependenciesTree capability lights up a Dependencies tree node and it's sub node providers-->
     <ProjectCapability Include="DependenciesTree" />
     <ProjectCapability Include="LaunchProfiles" />
-
+    <ProjectCapability Include="NoGeneralDependentFileIcon"/>
```

Comment:
Can you please update https://github.com/Microsoft/VSProjectSystem/blob/master/doc/overview/project_capabilities.md